versions/3.0.md

```diff
@@ -50,8 +50,8 @@ The HTTP Status Codes are used to indicate the status of the executed operation.
 
 ### Format
 
-The files describing the RESTful API in accordance with the Swagger specification are represented as JSON objects and conform to the JSON standards. YAML, being a superset of JSON, can be used as well to 
-represent a Swagger specification file.
+The files describing the RESTful API in accordance with the Open API Specification are represented as JSON objects and conform to the JSON standards. YAML, being a superset of JSON, can be used as well to 
```

Comment:
Seems like this space in `Open API Specification` is intentional, but it will be the only one in the doc, so it was worth mentioning as well.